transformers4rec/torch/utils/data_utils.py

```diff
@@ -72,6 +77,11 @@ def __init__(
             **kwargs,
         ):
             T4RecDataLoader.__init__(self)
+            warnings.warn(
```

Comment:
Merlin data loader is now supporting CPU-based loading as well, so we might want to drop the `pyarrow` data loader in future release?